trio/_core/_multierror.py

```diff
@@ -415,7 +415,7 @@ def traceback_exception_init(
         self.embedded = []
 
 
-traceback.TracebackException.__init__ = traceback_exception_init
+traceback.TracebackException.__init__ = traceback_exception_init  # type: ignore
```

Comment:
I'm not too worried about `# type: ignore` comments inside this horrible blob of monkeypatching, but I am curious why you need them.

Reply:
The error is `trio/_core/_multierror.py:418: error: Cannot assign to a method`. It appears to be a known mypy bug: https://github.com/python/mypy/issues/2427. It's unclear to me if `setattr` is a better workaround or not.